fiducialMarker/fiducialMarkerTest.py

```diff
@@ -0,0 +1,40 @@
+from cv2 import aruco
+from fiducialMarker import FiducialMarker
+import numpy
+import unittest
+
+
+class TestFiducialMarker(unittest.TestCase):
+
+    @staticmethod
+    def test_positive_generate_marker():
+        # img_true attained from output in python3 shell of following inputs:
+        # dictionary = aruco.getPredefinedDictionary(aruco.DICT_4X4_50)
+        # img_true = aruco.drawMarker(dictionary, 0, 6)
+        img_true = numpy.array([[0,   0,   0,   0,   0,   0],
+                                [0, 255,   0, 255, 255,   0],
+                                [0,   0, 255,   0, 255,   0],
+                                [0,   0,   0, 255, 255,   0],
+                                [0,   0,   0, 255,   0,   0],
+                                [0,   0,   0,   0,   0,   0]])
+        img_test = FiducialMarker.generate_marker(0, 6)

```

Comment:
```python
img_test = FiducialMarker.generate_marker(id=0, side_pixels=6) 
# hard to understand the parameters being passed in the test, this could help.
```